### PR TITLE
Migration to package quickchart endpoint

### LINKED ISSUE #1: Migration to package QuickChart endpoint
This task is set to look at the already standardized QuickChart endpoint from to package, taking into account the objectives that are stated in this[ document](https://docs.google.com/document/d/19J_cQiY-fAYI4k9VcRrn2oKkQxy8tqEWIaaPIF-avmQ/edit#).

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull-request for issue #[15104](https://github.com/slingr-stack/platform/issues/15104) created by @agreggio-slingr 

## What it does?

Create the Quickchart package

## How to test it?

You can install the following test package with the following credentials.

ssh: git@github.com:slingr-stack/stripe-package-test.git

You can use and modify the following script or use the documentation as a reference to perform tests

```js
 log(JSON.stringify(pkg.quickchart.api.chart.post({
        "backgroundColor": "white",
        "width": "500",
        "height": "300",
        "format": "png",
        "chart": {
          "type": "bar",
          "data": {
            "labels": [2012, 2013, 2014, 2015, 2016],
            "datasets": [{
              "label": "LABEL A",
              "data": [12, 6, 5, 18, 12]
            }, {
              "label": "LABEL B",
              "data": [4, 8, 16, 5, 5]
            }]
          }
      }
  })));
```

## Implementation notes

N/A

### CODE REVIEW

File: scripts/helpersDocs.md
Comment (pasaperez-work) [inline scripts/helpersDocs.md:1]: You can delete this file, it is no longer necessary

File: package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "name": "quickchart",
+    "label": "Quickchart Package",
+    "dependencies": [
+        {
+            "type": "service",
+            "name": "http",
+            "version": "v1.3.3",
```
Comment (pasaperez-work) [inline package.json]:
```suggestion
            "version": "v1.3.4",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            "label": "QR Response",
+            "name": "qrResponse"
+        }
+    ],
+    "metadata": [
+        {
+            "type": "script",
+            "namespace": "api",
+            "path": "/scripts/functions.js"
```
Comment (pasaperez-work) [inline package.json]:
```suggestion
            "path": "/scripts/api.js"
```

Comment (pasaperez-work) [reply package.json]: Rename the functions.js file

File: package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        },
+        {
+            "type": "flowStep",
+            "namespace": "steps",
```
Comment (pasaperez-work) [inline package.json]:
```suggestion
            "namespace": "apiCallQuickchart",
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            "path": "/flowSteps/apiCallQuickchart"
+        },
+        {
+            "type": "flowStep",
+            "namespace": "steps",
```
Comment (pasaperez-work) [inline package.json]:
```suggestion
            "namespace": "generateChartQuickchart",
```

File: package.json [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            "path": "/flowSteps/generateChartQuickchart"
+        },
+        {
+            "type": "flowStep",
+            "namespace": "steps",
```
Comment (pasaperez-work) [inline package.json]:
```suggestion
            "namespace": "generateQRQuickchart",
```

File: flowSteps/apiCallQuickchart/step.js [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+	var headers = options.headers || {};
+	headers = mergeJSON(headers, {"Content-Type": "application/json"});
+
+	options.headers = headers;
+	return options;
+}
+
+
+function mergeJSON (json1, json2) {
+	const result = {};
```
Comment (pasaperez-work) [inline flowSteps/apiCallQuickchart/step.js]: Do not use const on flow steps scripts, nor any ES6 feature

Comment (pasaperez-work) [inline flowSteps/apiCallQuickchart/step.js]:
```suggestion
	var result = {};
```

**Review by angel-perez-itti — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -0,0 +1,610 @@
+<table>
```
Comment (pasaperez-work) [inline README.md]:
```suggestion
<table class="table" style="margin-top: 10px">
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,610 @@
...
+    </thead>
+    <tbody>
+    <tr>
+        <td>Quickchart package</td>
+        <td>November 21, 2023</td>
+        <td>Detailed description of the API of the Quickchart package.</td>
+    </tr>
+    </tbody>
+</table>
+---
```
Comment (pasaperez-work) [inline README.md]: Use only one table, the one above

File: package.json (hunk 2 above)
Comment (pasaperez-work) [reply package.json]: Or if you prefer we can use functions.js for special helpers and api only for the default http methods.

**Review by angel-perez-itti — APPROVED** [review]
Looks good Ale, thank you for the changes!

**Pull request merged**